packages/components/src/color-palette/index.js

```diff
@@ -112,19 +112,26 @@ function MultiplePalettes( {
 	);
 }
 
-export function CustomColorPickerDropdown( { isRenderedInSidebar, ...props } ) {
+export function CustomColorPickerDropdown( {
+	isRenderedInSidebar,
+	popoverProps: receivedPopoverProps,
+	...props
+} ) {
+	const popoverProps = {
+		__unstableShift: true,
+		...( isRenderedInSidebar
+			? {
+					placement: 'bottom-start',
+					offset: 20,
```

Comment:
@talldan , do you think that the offset is working as expected here (when editing gradients in global styles in the site editor) ?

Reply:
What are you seeing? Something that tripped me up initially is that `Dropdown` has a default offset of `13`, so `20` is only offsetting by `7` from the default position.

The popover is using the `components-custom-gradient-picker__markers-container` element as the anchor/reference element, so the positioning looks correct (checked by passing `0` for the offset prop) and expecting the reference elements bounding box. I'm not sure how it ends up with that as the reference element.

I tried `200` for an offset and it was about 200 pixels below the reference, which is to be expected with the `bottom` placement—the main axis will be the direction of offset.